lib/client-assets.php

```diff
@@ -276,6 +276,8 @@ function gutenberg_register_scripts_and_styles() {
 		)
 	);
 
+	wp_enqueue_script( 'wp-block-directory' );
```

Comment:
I'm not sure where is the best place to enqueue this script. We enqueue styles here as well, so I bet it's fine to have them grouped together.

I noticed also this:
https://github.com/WordPress/gutenberg/blob/a56daebec30a3740116e7e495d9776ef4c3d51a6/lib/widgets-page.php#L110-L113

It might be a good idea to introduce one script handle which consolidates all such extensions and is filterable so every plugin would use this filter to register their own UI plugins, blocks and formatting libraries. This way it would be also very simple to get rid of the block directory if someone doesn't want to have it enabled.

Reply:
I'd have used the `enqueue_block_editor_assets` hook I think

Reply:
As noted in https://github.com/WordPress/gutenberg/pull/17576/files#r331999772, this is broken at the moment. I'm going to try to come up with a proper fix in WP core first.